### PR TITLE
remediations - make existing-or-new the first step of the wizard

### PR DESCRIPTION
**Pull request by jharting**
screenshot_20181213_174153

### CODE REVIEW

File: src/SmartComponents/Remediations/steps/ExistingOrNew.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        const { name, isNew, existing } = this.state;
+
+        return (
+            <React.Fragment>
+                <h1>Do you want to modify an existing Remediation or create a new one?</h1>
+                <Form>
+                    <div>
+                        <label>
+                            <input
+                                type="radio"
```
Comment (ryelo) [inline src/SmartComponents/Remediations/steps/ExistingOrNew.js]: Let's use the Patternfly inputs:
https://patternfly-react.surge.sh/patternfly-4/components/radio

File: src/SmartComponents/Remediations/steps/ExistingOrNew.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                                name="radio"
+                                value="existing"
+                                checked={ !isNew }
+                                disabled={ !existing || !existing.length }
+                                onChange={ () => this.onRadioChange(false) }
+                            />
+                            Existing Remediation
+                        </label>
+
+                        <select disabled={ isNew || !existing } onChange={ this.onRemediationSelected } >
```
Comment (ryelo) [inline src/SmartComponents/Remediations/steps/ExistingOrNew.js]: Select: https://patternfly-react.surge.sh/patternfly-4/components/select

**Comment by ryelo** [comment]
@jharting Reread our slack conversation, there are patterns for individual inputs (radio, select, dropdown), but there is no pattern for combining the radio and select on one line. Sorry for the confusion.

**Comment by jharting** [comment]
@ryelo Oops! Well, my bad for living under a rock. Anyway, I switched to PF radios and selects. Please carefully review especially CSS and class names.

screenshot_20181214_094324

**Pull request merged**